pallets/poa/src/lib.rs

```diff
@@ -990,17 +992,17 @@ impl<T: Trait> Module<T> {
         expected_slots_per_validator: EpochLen,
         slots_per_validator: EpochLen,
         validator_block_counts: BTreeMap<T::AccountId, EpochLen>,
-    ) -> u128 {
-        let mut total_validator_reward = 0u128;
+    ) -> Balance {
+        let mut total_validator_reward = 0 as Balance;
         // Maximum emission per validator in this epoch
         let max_em = Self::get_max_emission_reward_per_validator_per_epoch(
             expected_slots_per_validator,
             slots_per_validator,
         );
-        let lock_pc = Self::validator_reward_lock_pc() as u128;
+        let lock_pc = Self::validator_reward_lock_pc() as Balance;
         for (v, block_count) in validator_block_counts {
             // The actual emission rewards depends on the availability, i.e. ratio of blocks produced to slots available
-            let reward = max_em.saturating_mul(block_count.into()) / (slots_per_validator as u128);
+            let reward = max_em.saturating_mul(block_count.into()) / (slots_per_validator as Balance);
 
             /*print(reward as u64);
```

Comment:
Nit: Block comment out of date. Suggest removal.

Reply:
Removed